apps/api/src/server/migration/migrators/nsip-exam-timetable-migrator.js

```diff
@@ -207,9 +213,9 @@ const formatBulletPoints = (input) => input.trim().replace(/(\r\n[?•-])/g, '\r
  */
 const isDeadlineType = (examinationTimetableItemType) => {
 	const examinationItemDeadlineTypes = [
-		'deadline',
-		'deadline for close of examination',
-		'procedural deadline (pre-examination)'
+		examTimetableItemTypes.DEADLINE,
+		examTimetableItemTypes.DEADLINE_FOR_CLOSE_OF_EXAMINATION,
+		examTimetableItemTypes.PROCEDURAL_DEADLINE
 	];
-	return examinationItemDeadlineTypes.includes(examinationTimetableItemType.toLowerCase());
+	return examinationItemDeadlineTypes.includes(examinationTimetableItemType);
```

Comment:
should we do some sort of string manipulation to ensure that the capitalisation doesn't matter here? I can see that the `examTimetableItemTypes` constant has the first letter capitalised, just unsure if we can guarantee that the types coming from Hzn will also be in this format